niworkflows/anat/ants.py

```diff
@@ -979,11 +972,9 @@ def init_n4_only_wf(
     try:
         inu_n4_final.inputs.rescale_intensities = True
     except ValueError:
-        warn(
-            "The installed ANTs version too old. Please consider upgrading to "
-            "2.1.0 or greater.",
-            DeprecationWarning,
-        )
+        warn("N4BiasFieldCorrection's --rescale-intensities option was added in ANTS 2.1.0 "
+             f"({inu_n4_final.interface.version} found.) Please consider upgrading.",
+             DeprecationWarning)
```

Comment:
`DeprecationWarning`s are for things we plan to deprecate. I think this should be a `UserWarning`.